src/alpha-lab/README.md

```diff
@@ -130,6 +130,12 @@ export default {
           to: "/alpha-lab/gaze-contingency-assistive",
           img: "gaze-cont-main-page.png",
         },
+        {
+          title: "Blink and you'll miss it!",
+          text: "Run our blink detection pipeline offline and/or in real-time with Neon",
+          to: "/alpha-lab/blink-detection",
+          img: "eye_blinks_anim.gif",
```

Comment:
I also selected the gif to use as a preview, but it doesn't format correctly. Creating a gif of a single eye will probably be the most efficient solution.